api/config.py

```diff
@@ -74,12 +74,13 @@ class Config(object):
     # You can disable NRO updates for Name Requests by setting the variable in your .env / OpenShift configuration
     DISABLE_NAMEREQUEST_NRO_UPDATES = int(os.getenv('DISABLE_NAMEREQUEST_NRO_UPDATES', 0))
     DISABLE_NAMEREQUEST_SOLR_UPDATES = int(os.getenv('DISABLE_NAMEREQUEST_SOLR_UPDATES', 0))
+    DISABLE_NAMEREQUEST_NATS_UPDATES = int(os.getenv('DISABLE_NAMEREQUEST_NATS_UPDATES', 0))
 
     # NATS
-    NATS_SERVERS = os.getenv('NATS_SERVERS')
-    NATS_CLIENT_NAME = os.getenv('NATS_CLIENT_NAME')
-    NATS_CLUSTER_ID = os.getenv('NATS_CLUSTER_ID')
-    NATS_QUEUE = os.getenv('NATS_QUEUE')
+    NATS_SERVERS = os.getenv('NATS_SERVERS', 'nats://localhost:4222')
```

Comment:
Are you sure it's OK to commit any or all of these?

Reply:
Yes, they are just default values to help other developers to configure it.

Reply:
This conversation can be closed. (I do not have permission to do so.)